dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -901,7 +901,7 @@ protected void addLabelForNextNode(Label l) {
   /* --------------------------------------------------------- */
 
   /** The UID for the next unique name. */
-  protected long uid = 0;
+  protected static long uid = 0;
```

Comment:
What's the reasoning for this change? On its surface, this seems like a bad idea: why would different CFGs need to share a uid counter?